html_app/ui/FacsView.js

```diff
@@ -616,11 +616,20 @@ scb.ui.static.FacsView.reevaluate_metadata = function (state) {
         pts.display_id= pts.display_id || new_id;
         pts.bisector_id = pts.bisector_id || bisector_id;
         pts.unique_id = pts.unique_id || Math.floor(Math.random()*1000000000).toString(27);
+
+        var scaled_from=pts.from;
+        var scaled_to=pts.to;
```

Comment:
A comment explaining why we are showing different numbers than we are graphing would be helpful for future generations